### ISSUE TITLE
Extension option on menu remains greyed out after installation

### ISSUE THREAD

**Issue by PhilvanKleur** [jwcliff/Inkscape_Exportxy#4]
This is on Inkscape 1.0 (4035a4fb49, 2020-05-01) on Windows 10.

I downloaded the zip file from here this morning, unzipped it, and copied the Inkscape_Exportxy-master directory into my extensions directory. I then closed and restarted Inkscape, which I'd had open. The extension's option got added as Extensions > Export > Export XY, but was greyed out. Restarting Inkscape made no difference. 

I tried changing the case of "X" and "Y" in the filenames to be consistent throughout, though since Windows ignores case in filenames, I didn't expect that to be a problem. It didn't help, anyway.

I don't know much about .inx files, but the extension's looks analogous to that of an extension I installed successfully, Apply Transforms. I can't see anything obviously wrong, given that similarity.

So I don't know what to do. Presumably Inkscape must "know" that some condition has failed, in order to grey out the menu option. Wouldn't it be helpful to report what that condition is, in English, in case it helps the user? E.g. "Manifest file not found", or "Python file locked against reading", or "Python can't find imported module".

I attach screenshots showing: the extension's directory in my extensions directory; the contents of the extension's directory; the .inx file, and the top of the Python file; the Edit > Preferences setting for my extensions directory, just to show that it matches where I put the extension; and the greyed-out menu option.

 
![Screenshot (242)](https://user-images.githubusercontent.com/68313465/97610599-02467100-1a0d-11eb-8b46-b3893f764644.png)
![Screenshot (244)](https://user-images.githubusercontent.com/68313465/97610601-03779e00-1a0d-11eb-8efa-6c2e75782f77.png)
![Screenshot (246)](https://user-images.githubusercontent.com/68313465/97610604-04103480-1a0d-11eb-816f-b34c0e164c15.png)
![Screenshot (247)](https://user-images.githubusercontent.com/68313465/97610609-04a8cb00-1a0d-11eb-84de-f5f77f28b101.png)
![Screenshot (248)](https://user-images.githubusercontent.com/68313465/97610612-05416180-1a0d-11eb-98a0-8bdbdc1992d6.png)

**Comment by Simarilius-uk** [comment]
Is it because you haven't got a selection?
I'll try take a look at it this weekend.

**Comment by PhilvanKleur** [comment]
No. I'd tried that, but I've just done so again. I selected the path whose coordinates I wanted and then called up the Extensions > Export menu, but the Export XY option remains greyed out. 

BUT, I am using an alternative fall-back method to get my coordinates, see the next paragraph, and since opening the issue, I found that this provokes a hint. I call Inkscape from the command line. When I do this, it says:
** (org.inkscape.Inkscape:33564): WARNING **: 02:29:05.242: Failed to find resource file 'ExportXY.py'. Looked in:
        C:\Users\Me\AppData\Roaming\inkscape\extensions\ExportXY.py
        c:\Program Files\Inkscape\share\inkscape\extensions\ExportXY.py
Any ideas? I thought that ExportXY.py , being part of the extension's code, would be expected to be in a subdirectory, not in the top-level extensions directory. 

Incidentally, I'm using this because I'm trying to make animations move along a curve, so I need the curves' coordinates. I can fudge it by manually placing circles along the curves, then getting their coordinates with  -S from the Inkscape command line, and then fitting a polynomial to those. It works reasonably well with a fourth-order polynomial, doing the fitting in R with the `lm()` function. But it's less convenient, and must be somewhat less accurate.

**Comment by AlfredoRates** [comment]
I had the same issue. I am using Inkscape 1.3.1. What worked for me was to edit the .inx file a bit, based on other extensions. I am new in Inkscape so can't explain why it worked, I guess new version.

1. comment/erase lines 5 and 6
2. change line 7 to simply "<effect>"
3. In line 14, change "reldir" with "location"
